packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -254,8 +254,11 @@ class GameWidgetState<T extends Game> extends State<GameWidget<T>> {
     _loaderFuture = null;
   }
 
-  void disposeCurrentGame() {
+  void disposeCurrentGame({bool isDispose = false}) {
```

Comment:
`isDispose` needs a clearer name for what it does. In which cases will this method be called with this variable set to false?

Reply:
Changed to `callGameOnDispose`.  I think that is pretty straightforward now.

Reply:
I still think it is a bit unclear, especially since this method is called `disposeCurrentGame`, the reader of the code would have to guess in what cases such a variable would be false.
But maybe it's fine if we just get some dartdocs for this method explaining how that flag is used.

Reply:
I updated the diagram and the dart doc in the comments above.

Reply:
I can't see the dartdoc being updated? :thinking: 
![image](https://github.com/flame-engine/flame/assets/744771/0ccb9528-42e4-45ea-a45c-bf6cf20ea8de)

Reply:
Look again, lol.  Forgot that part.